llvm/include/llvm/Support/TimeProfiler.h

```diff
@@ -83,16 +83,27 @@ namespace llvm {
 
 class raw_pwrite_stream;
 
+struct TimeTraceMetadata {
```

Comment:
Could we also include the line numbers?
I suspect it would be trivial to add that, but we will have an option of pointing at an exact line in the file.
It should greatly improve the UX of the tools that are capable of consuming this information.

Reply:
Done.